app/views/attendances/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<p style="color: green"><%= notice %></p>
```

Comment:
also here, you could create a class with this property and use it

Reply:
changed